README.md

```diff
@@ -25,50 +25,105 @@ and start contributing!
 
 *The API is in a very early stage. Everything is subject to change.*
 
-### /ec2 ###
-
-Search EC2 instances.
-
-Optional parameters:
+### Optional parameters ###
+The `ec2`, `dynamo`, `sqs` and `elasticache` queries support following optional parameters:
+- `q` / `query`: OGNL expression used to filter. Example:
+  - `id>100&&readCapacityUnits>10`
+- `s` / `sort`: OGNL expression used as a ``Comparable`` to sort (default to no ordering). Example:
+  - `s=itemCount`
+- `l` / `limit`: maximum number of records to return. Example:
+  - `l=10`
+- `f` / `field`: comma-separated list of fields to display(defaults to all). Examples:
+  - `f=tableName,tableStatus`
+  - `f=tableName`
 
-- `q`/`query`: OGNL expression used to filter (defaults to all instances). Examples:
+Sample query: show readCapacityUnits and tableName of all tables sorted by readCapacity:
+``/dynamo?s=readCapacityUnits&&f=readCapacityUnits,tableName&&l=3``
 
-  - `state=="running"&&key=="pierre"&&launchTime>1365000000000`
+Sample response:
+```
+[
+  {
+    tableName: "table_foo",
+    readCapacityUnits: 1
+  },
+  {
+    tableName: "table_bar",
+    readCapacityUnits: 2
+  },
+  {
+    tableName: "table_baz",
+    readCapacityUnits: 3
+  }
+]
+```
 
-- `s`/`sort`: OGNL expression used as a `Comparable` to sort (default to no ordering).
-Example: `daysOld`.
+### EC2 Query ###
 
-- `l`/`limit`: maximum number of records to return (defaults to none). Example: `10`.
+#### /ec2/sg ####
+List all ec2 security groups.
 
-- `f`/`fields`: comma-separated list of fields to display (defaults to all). Examples:
+#### /ec2/all ####
+List all ec2 instances.
```

Comment:
It's a bit silly to have both `/ec2/all` and `/ec2` without optional params return the exact same result. That said, if we shore this up there is some chance that we break backward compatibility, so I'm okay to keep it.